### ISSUE TITLE
Script issues - support for UDMPro v2.4.23

### ISSUE THREAD

**Issue by mojo333** [alxwolf/ubios-cert#32]
1. `deploy.sh` fails for me as I get the following when executing:
```
root@UDMP:/data/letsencrypt/ubios-cert-main# ./deploy.sh
Unsupported model: UniFi Dream Machine Pro
```
Adding an echo just before the case statement shows that ${MODEL} evaluates to just `"UniFi"` and hence why the case statement fails... something to do with spaces in the variable. Not sure how to fix this as I'm no bsh expert.

2. `ubios-cert.sh` also needs updated code to handle UDMPro 2.4.23 as was done for `deploy.sh` - see excerpt below:

```
# When running in UDM SE / UDR, the applications are not running in containers
case "$(ubnt-device-info model || true)" in "UniFi Dream Machine SE"|"UniFi Dream Router")
	# But here, the same command will be executed but inside unifi-os container
	IS_UNIFI_2='true'
	;;
esac
```

**Comment by ausil** [comment]
I have submitted a fix in PR #33 sorry I must have missed that bit when testing it late last night

**Comment by mojo333** [comment]
> I must have missed that bit when testing it late last night

I'm keen to get this working, so thanks for your support on this.
